docs/quickstart.html

```diff
@@ -46,6 +46,14 @@ <h4 class="anchor-heading">
             NOTE: Your local environment must have Java 8+ installed.
         </p>
 
+        <p>
+            Apache Kafka can be started using ZooKeeper or KRaft. To get started with either configuration follow one the sections below but not both.
+        </p>
+
+        <h5>
+            Kafka with ZooKeeper
+        </h5>
+
         <p>
             Run the following commands in order to start all services in the correct order:
         </p>
```

Comment:
Should we also remove the comment in zookeeper start command: (L62)
```
<pre class="line-numbers"><code class="language-bash"># Start the ZooKeeper service
[# Note: Soon, ZooKeeper will no longer be required by Apache Kafka.] <== remove this since it already happened
$ bin/zookeeper-server-start.sh config/zookeeper.properties</code></pre>
```

Reply:
Done.